### ISSUE TITLE
vk.xml v1.0.45, no value specified for sType field of struct VkPhysicalDeviceFeatures2KHR

### ISSUE THREAD

**Issue by ParticlePeter** [KhronosGroup/Vulkan-Docs#480]
I guess this is not intentional, but not sure. I think vk.xml line 1770 f:
```
        <type category="struct" name="VkPhysicalDeviceFeatures2KHR">
            <member><type>VkStructureType</type> <name>sType</name></member>
```
was and should be this way:
```
        <type category="struct" name="VkPhysicalDeviceFeatures2KHR">
            <member values="VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_2_KHR"><type>VkStructureType</type> <name>sType</name></member>
 ```

**Comment by krOoze** [comment]
Consequentially the spec says:

> `sType` **must** be VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_KHR

Which is a non-existent enumerant (missing the "2" -- not sure how the validity generator came up with that name). Definitely some kind of bug.

<hr />
`values` are also missing for `VkValidationFlagsEXT`, if not intentional.
<hr />

(
BTW there is some naming inconsistency: `VK_STRUCTURE_TYPE_SURFACE_CAPABILITIES2_EXT` vs `VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_2_KHR` ("X_2_" vs "X2_").
)

**Comment by oddhack** [comment]
This should be fixed in the 1.0.46 spec update.
